x-pack/legacy/plugins/infra/common/http_api/log_analysis/validation/indices.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+
+export const LOG_ANALYSIS_VALIDATION_INDICES_PATH = '/api/infra/log_analysis/validation/indices';
+
+/**
+ * Request types
+ */
+export const validationIndicesRequestPayloadRT = rt.type({
+  data: rt.type({
+    timestamp: rt.string,
+    indices: rt.string,
```

Comment:
We expect the errors to be returned for each comma-separated segment individually. Would it be more consistent to also supply them as an array?